stable/velero/templates/deployment.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if .Values.configuration.provider -}}
+{{- $provider := .Values.configuration.provider -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "velero.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "velero.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "velero.chart" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/name: {{ include "velero.name" . }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "velero.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        helm.sh/chart: {{ include "velero.chart" . }}
+    {{- if or .Values.podAnnotations .Values.metrics.enabled }}
+      annotations:
+      {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.metrics.enabled }}
```

Comment:
Noticed this while testing, if you enable metrics, it will template `true` into the annotations. I guess this should be `{{- with .Values.metrics.podAnnotations }}`?

Reply:
Done